<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 7pt &quot;Arial&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_c">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Topwear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bottomwear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shoes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bags</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Watches</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Innerwear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eyewear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jewellery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fragrance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Others</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_s">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Summer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Winter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spring</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fall</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_g">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Men</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Women</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_c">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>category</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_s">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>season</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_g">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_description">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:8pt;&quot;&gt;input your description here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit2">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>260</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt;&quot;&gt;:) message box&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="image3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>60</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../docs/empty.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH MODE 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH MODE 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE DISPLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changebutton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 6pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CHANGE PHOTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="id1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> id:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit1">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changebutton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 6pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CHANGE PHOTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 203, 255);</string>
    </property>
    <property name="text">
     <string>Go To Image Classification</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSearch_Mode">
    <property name="title">
     <string>Search Mode</string>
    </property>
   </widget>
   <addaction name="menuSearch_Mode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
